rest_framework_nested/relations.py

```diff
@@ -9,6 +9,40 @@
 import rest_framework.relations
 
 
+class NestedHyperlinkedIdentityField(rest_framework.relations.HyperlinkedIdentityField):
```

Comment:
If this is going into master, we'll probably want to deal with these `FIXME` notes and test them?